### ISSUE TITLE
Community Description - revision request

### ISSUE THREAD

**Issue by seanluyk** [ualbertalib/jupiter#1907]
The creator of the [Workspace Canada](https://era.library.ualberta.ca/communities/ec2674fb-bb6a-47bb-8d47-c29c40e98bb1) materials has requested we modify the current community description. He's provided us with a new description that includes formatting: 
- [WSC-ERA-Summary-New.docx](https://github.com/ualbertalib/jupiter/files/5315194/WSC-ERA-Summary-New.docx)

I'm assuming we can't format text in the description, but is it possible to edit it?

**Comment by mbarnett** [comment]
Formatting isn't supported at the moment. I'll take a closer look when I'm back from vacation next Wednesday, but I'm 99% certain that we're not going to be able to modify this until after the migration.

**Comment by seanluyk** [comment]
Thanks @mbarnett, no worries!

**Comment by seanluyk** [comment]
@mbarnett just checking in on this as the user has asked if this could be edited soon. I'm assuming no but I thought I would check

**Comment by mbarnett** [comment]
Hey Sean, sorry yes this is blocked by the issues with Fedora Communities. Hard to give an exact timeline of when we'll be through the migration to unblock this given the SET stuff, but I guess we can let them know that this is currently frozen due to some upgrades that are in progress and we're trying to make the infrastructure changes necessary to unblock this ASAP?

**Comment by seanluyk** [comment]
Thanks @mbarnett I'll let them know

**Comment by mbarnett** [comment]
This is also now unblocked (the description editing, anyways, no new support for formatting)

**Comment by mbarnett** [comment]
Closing since this seems to be fixed, but please reopen (or open a new issue, no real preference) if you hear from the user that they're still experiencing issues.
